data/tx/retired/Drew-Springer-e1684e74-28bf-483e-ab9b-7eaacc8cd868.yml

```diff
@@ -0,0 +1,21 @@
+id: ocd-person/e1684e74-28bf-483e-ab9b-7eaacc8cd868
+name: Drew Springer
+given_name: Drew
+family_name: Springer
+image: https://house.texas.gov/members/photos/4075.jpg?v=88.6
+party:
+- name: Republican
+roles:
+- start_date: '2021-03-09'
+  end_date: '2023-01-11'
+  type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:tx/government
+  district: '68'
+- end_date: '2023-01-11'
+  type: upper
+  jurisdiction: ocd-jurisdiction/country:us/state:tx/government
```

Comment:
@sroomf this file can be deleted. this person should already have a file, their roles should look like:

- end_date: '2021-01-06'
  type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:tx/government
  district: '68'
- start_date: '2021-01-06'
  type: upper
  jurisdiction: ocd-jurisdiction/country:us/state:tx/government
  district: '30'